### DISCUSSION TITLE
Some installation guidance....

### DISCUSSION THREAD

**Discussion by tall-vish** [winapps-org/winapps#318] [Support]
I have followed the steps mentioned on the home page but a few caveats:

1.  when running the `bash <(curl https://raw.githubusercontent.com/winapps-org/winapps/main/setup.sh)` without sudo, I get the attached error. Actually, wasn't able to do anything without su. Is that a problem?
2.  After all the song and dance, how do I install photoshop? There is no information on the guide about this
3. I was able to launch the setup.sh by downloading it and putting the inquirer file in the winapps folder. Still can't run it outside su.

It seems like, thanks to me running through the entire process using su, it is looking for files in the root folder

Thanks for your help, please be advised that I am a noob in linux, using openSuSE tumbleweed.
![Screenshot_20241108_154430](https://github.com/user-attachments/assets/3246000c-a53f-4d4f-bd0d-d9cd52aea88f)

**Comment by oskardotglobal** [comment]
Can you try running it like `bash <(curl https://raw.githubusercontent.com/winapps-org/winapps/main/setup.sh) --user` without sudo?

**Reply by tall-vish** [reply]
getting the same error using that.

**Reply by tall-vish** [reply]
reinstalled the container without using sudo.
No winapps-src folder in the .local/bin location, am I supposed to make the folder?
Getting same error when running curl command

**Comment by AlphaZulu666** [comment]
Just do: rm -rf .local/bin/winapps-src/
then retry...
It looks like the script does not clean up after giving an error... or just can't recover if it has left garbage there :)

**Reply by tall-vish** [reply]
Tried that, giving me the same error. Should I try reinstalling the entire thing? Can it be done without using `sudo`?

**Reply by tall-vish** [reply]
reinstalled the container without using `sudo`.
No winapps-src folder in the .local/bin location
Getting same error when running curl command
